mlrun/api/crud/model_endpoints.py

```diff
@@ -104,9 +104,11 @@ def create_or_patch(
                 model_endpoint.spec.monitoring_mode
                 == mlrun.api.schemas.ModelMonitoringMode.enabled.value
             ):
-                self.create_monitoring_feature_set(
+                m_fs = self.create_monitoring_feature_set(
```

Comment:
let's try to have params with explicit names
```suggestion
                monitoring_feature_set = self.create_monitoring_feature_set(
```